scripts/train.ipynb

```diff
@@ -1,71 +1,170 @@
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 2,
- "cells": [
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
+    "cellView": "form",
+    "id": "tc8tY74IY8sA"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Colab Shell\n",
+    "%%html\n",
+    "<div id=term_demo></div>\n",
+    "<script src=\"https://code.jquery.com/jquery-latest.js\"></script>\n",
+    "<script src=\"https://cdn.jsdelivr.net/npm/jquery.terminal/js/jquery.terminal.min.js\"></script>\n",
+    "<link href=\"https://cdn.jsdelivr.net/npm/jquery.terminal/css/jquery.terminal.min.css\" rel=\"stylesheet\"/>\n",
+    "<script>\n",
+    "  $('#term_demo').terminal(async function(command) {\n",
+    "      if (command !== '') {\n",
+    "          try {\n",
+    "              let res = await google.colab.kernel.invokeFunction('shell', [command])\n",
+    "              let out = res.data['application/json'][0]\n",
+    "              this.echo(new String(out))\n",
+    "          } catch(e) {\n",
+    "              this.error(new String(e));\n",
+    "          }\n",
+    "      } else {\n",
+    "          this.echo('');\n",
+    "      }\n",
+    "  }, {\n",
+    "      greetings: 'Welcome to Colab Shell',\n",
+    "      name: 'colab_demo',\n",
+    "      height: 250,\n",
+    "      prompt: 'colab > '\n",
+    "  });"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "W-1VlSbDdG5o"
+   },
+   "outputs": [],
+   "source": [
+    "# MOUNT GDRIVE TO COLAB\n",
+    "IN_COLAB = 'google.colab' in str(get_ipython())\n",
+    "if IN_COLAB:\n",
+    "  from google.colab import drive\n",
+    "  drive.mount('/content/drive')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-iy0eKtCh-lK"
+   },
+   "source": [
+    "# For all Jupyter-like environments"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "ml0Cehwnbtfk"
+   },
+   "outputs": [],
+   "source": [
+    "# UNCOMENT REQUIREMENTS TO BE INSTALLED\n",
+    "# from IPython.display import clear_output\n",
+    "# !pip install -r requirements_colab.txt\n",
+    "# !pip install -r requirements.txt\n",
+    "# clear_output(wait=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "P8ic_dvAYmGe",
     "tags": []
    },
    "outputs": [],
    "source": [
     "%matplotlib inline\n",
     "%load_ext autoreload\n",
-    "%autoreload 2"
+    "%autoreload 2\n",
+    "%config IPCompleter.greedy=True"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "PzRmhaqyYmGe"
+   },
    "outputs": [],
    "source": [
-    "# CHECK WORKIND DIR PATH\n",
+    "# MODIFY LIBRARY PATH\n",
     "import os\n",
-    "# MODIFY SYSTEM PATH\n",
-    "os.chdir('../')\n",
-    "os.getcwd()"
+    "from pathlib import Path\n",
+    "lib_path = Path('/Users/igorgarbuz/Dev/catlearn')\n",
```

Comment:
Use a relative path, and use the resolve method on the Path object to get the absolute path

Reply:
relative path imposes a constrain on from where you start jupyter. user may not be sure about lib relative location till he runs os.getcwd(). propose to keep it absolute.

Reply:
The way it is set now is specific to the environment it is used in, which is worse. So I still think it's better with a relative path.

Reply:
ok, agree it isn't ok with personal Path. put relative instead with a small check and advice to replace by absolute instead